### ISSUE TITLE
What is the correct BatchNorm layer for digits?

### ISSUE THREAD

**Issue by RSly** [NVIDIA/DIGITS#1433]
Hi,

I would like to know what is the correct way of defining BatchNorm with the latest Digits 5.1-dev and NVcaffe 0.15.14

I have been trying the suggestions in [#629](https://github.com/NVIDIA/DIGITS/issues/629) and [#3347](https://github.com/BVLC/caffe/issues/3347) with little success.

**Comment by lukeyeager** [comment]
If it helps, here are the parameters you can set for batchnorm in NVcaffe 0.15.14:
https://github.com/NVIDIA/caffe/blob/v0.15.14/src/caffe/proto/caffe.proto#L587-L605

As far as I know, this example should have the correct batchnorm settings for that version of NVcaffe:
https://github.com/NVIDIA/caffe/blob/v0.15.14/examples/cifar10/cifar10_full_sigmoid_train_test_bn.prototxt#L68-L86

**Comment by RSly** [comment]
thanks for your answer @lukeyeager!

unfortunately batch normalization with deep networks +digits/nvcaffe is very confusing.

also no one has commented on these issues for nvcaffe regarding bugs and questions since weeks...
see 
https://github.com/NVIDIA/caffe/issues/279

and

https://github.com/NVIDIA/caffe/issues/276

I just wonder if someone has a working template?
